sigs.yaml

```diff
@@ -2766,6 +2766,41 @@ workinggroups:
   contact:
     slack: wg-policy
     mailing_list: https://groups.google.com/forum/#!forum/kubernetes-wg-policy
+- dir: wg-reliability
+  name: Reliability
+  mission_statement: >
+    Allow users to safely use Kubernetes for managing production workloads by ensuring
```

Comment:
`Allow users to safely use Kubernetes for managing production workloads by ensuring Kubernetes is stabie and reliable.`

reads a little bit better. wdyt?

Reply:
done